### PR TITLE
Add php 8 support

### PR DESCRIPTION
**Pull request by GuilhemN**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.1|^8.0",
```
Comment (maxhelias) [inline composer.json:18]: We should use `>=7.1.3`

Comment (GuilhemN) [reply composer.json:18]: I have no strong opinion about this, both options are fine for me, so sure we can do that.

**Review by maxhelias — APPROVED** [review]

**Comment by GuilhemN** [comment]
Do you know why we're getting these failures on travis:
```
4) Nelmio\ApiDocBundle\Tests\Describer\RouteDescriberTest::testIgnoreWhenNoController

ParseError: syntax error, unexpected token "match", expecting variable

/home/travis/build/nelmio/NelmioApiDocBundle/Tests/Describer/RouteDescriberTest.php:32

5) Nelmio\ApiDocBundle\Tests\Functional\BazingaFunctionalTest::testModelComplexDocumentationBazinga

ParseError: syntax error, unexpected token "match"

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/willdurand/negotiation/src/Negotiation/Negotiator.php:41

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/config/Resource/ClassExistenceResource.php:84

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/dependency-injection/ContainerBuilder.php:348

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/dependency-injection/Dumper/PhpDumper.php:500

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/dependency-injection/Dumper/PhpDumper.php:1452

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/dependency-injection/Dumper/PhpDumper.php:1221

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/dependency-injection/Dumper/PhpDumper.php:241

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/http-kernel/Kernel.php:715

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/http-kernel/Kernel.php:543

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/http-kernel/Kernel.php:131

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/framework-bundle/Test/KernelTestCase.php:80

/home/travis/build/nelmio/NelmioApiDocBundle/vendor/symfony/framework-bundle/Test/WebTestCase.php:46

/home/travis/build/nelmio/NelmioApiDocBundle/Tests/Functional/BazingaFunctionalTest.php:22
```

We should at least make sure they only affect our test suite and are not also breaking the bundle in more normal situations.

**Comment by maxhelias** [comment]
@GuilhemN Yes, I already have this error and it's related to PhpUnit but I don't know why anymore.

**Comment by maxhelias** [comment]
Oh, i think this is the version defined here : https://github.com/nelmio/NelmioApiDocBundle/blob/master/phpunit

BTW, we should be able to do without this file with the symfony-bridge. Or was there a particular reason?

**Comment by GuilhemN** [comment]
Thanks!

I think it was for compatibility with lower php versions, I'm not sure whether this is still necessary, I launched the tests to check that.

**Comment by stayeronglass** [comment]
any news?

**Comment by jorge07** [comment]
willdurand/Negotiation needs update here the fail reason in CI: https://github.com/willdurand/Negotiation/pull/106

**Comment by GuilhemN** [comment]
Actually is due to FOSRestBundle, but it is a work in progress, see https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2280 and https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2278

**Comment by firoj07** [comment]
#1781

**Comment by firoj07** [comment]
My help you

**Pull request merged**